torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -82,7 +82,9 @@ class _X86InductorQuantizationAnnotation(QuantizationAnnotation):
 
 # A superset of default_quantizable_ops includes operators support the int8 data type
 # but not enabled by default recipe of X86InductorQuantizer.
-quantizable_ops = default_quantizable_ops
+quantizable_ops = default_quantizable_ops | {
```

Comment:
why not just add matmul to default_quantizable_ops?

Reply:
Because we think it's not a general recipe. `matmul` recipe has been used for Bert-Large and VIT, but not used for most of the LLM models because of the accuracy concern. So, we decide don't put it into the `default_quantizable_ops`.

Reply:
I mean what is the difference, since you are annotating it in the quantizer anyways?

Reply:
Oh, got it. Since `matmul` is not in `default_quantizable_ops`, `if config := self._get_aten_operator_type(torch.ops.aten.matmul.default):` will return None by default and we will not annotate `matmul` actually. Unless, we use `_set_aten_operator_type` to set the Quantization config for `matmul`.

Reply:
OK, I see. sounds good